tower-scripts/bin/cicd-control.sh

```diff
@@ -22,207 +23,252 @@ function on_exit() {
     fi
 }
 
-function is_running(){  
-  # Output to prevent ssh timeouts. Appears to timeout
-  # After about an hour of inactivity. 
-  while true; do
-    echo 
-    echo ".... $(date) ...."
-    sleep 600
-  done
-}
-
 trap on_exit EXIT
 
+function parse_and_set_vars() {
+  local var=$(echo $1 | awk -F= '{print $1}')
+  local value=$(echo $1 | awk -F= '{print $2}')
+
+  eval $var=\$value
```

Comment:
Need a list of acceptable -a keys. Unrecognized keys should result in an error in order to prevent typos.